### DISCUSSION TITLE
Digital Version of A Pattern Language by Christopher Alexander and co

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#445] [Ideas]
Have loved Christopher Alexander and co's A Pattern Language for more than 20 years. Finally have a chance to apply it in my own life in helping design a small hamlet in rural France.

Would love decent online version with search, directory of patterns, linking, real text etc

See https://twitter.com/rufuspollock/status/1646138887042134019

WIP at https://github.com/life-itself/pattern-language

**Comment by davidgasquez** [comment]
Came across this YouTube video showing a beautiful digital version of the patterns. 

https://youtu.be/UWCZUly2_2k

Couldn't find anything else so perhaps is only a sketch/demo.

**Reply by rufuspollock** [reply]
Nice find. I'm guessing it never got built.

**Reply by davidgasquez** [reply]
Gave another go at looking for existing resources and found [a website with the pattern list](https://claytondorge.com/patterns-list).

They're also publishing them to Twitter under the [`@apatterntolearn`](https://twitter.com/apatterntolearn) handle.

**Reply by rufuspollock** [reply]
Great sleuthing @davidgasquez - do you want to add these to https://github.com/life-itself/pattern-language/issues/1

**Reply by davidgasquez** [reply]
Sure! I didn't know about the issue. 😊

**Reply by rufuspollock** [reply]
That's because it just got opened 😜

**Comment by rufuspollock** [comment]
As an update: have a private repo atm with fully extracted text (via OCR). Looking for folks who would like to proof edit and more ...
